### ISSUE TITLE
Check for no-reraise-cancelled and cancelled-not-raised in `except*`

### ISSUE THREAD

**Issue by Zac-HD** [python-trio/flake8-async#311]
```python
try:
    ...
except* trio.Cancelled:
    pass
```
is just as bad as the `except:` form, and thus I think we should apply exactly the same lint rule to these cases.

**Comment by jakkdl** [comment]
was fixed in #358
